docs/mfc/tn035-using-multiple-resource-files-and-header-files-with-visual-cpp.md

```diff
@@ -47,17 +47,22 @@ This note is structured to answer the above questions as follows:
 
 Visual C++ manages a single .RC resource file and a corresponding .H header file as a tightly coupled pair of files. When you edit and save resources in an .RC file, you indirectly edit and save symbols in the corresponding .H file. Although you can open and edit multiple .RC files at a time (using Visual C++'s MDI user interface) for any given .RC file you indirectly edit exactly one corresponding header file.
 
-### Symbol Header File
+### Resource View's Resource Includes dialog
 
-By default, Visual C++ always names the corresponding header file RESOURCE.H, regardless of the name of the resource file (e.g., MYAPP.RC). Using the **Resource Includes** command from the **View** menu in Visual C++, you can change the name of this header file by updating the Symbol Header File file in the **Set Includes** dialog box.
+#### Symbol Header File
 
-### Read-Only Symbol Directives
+By default, Visual C++ always names the corresponding header file RESOURCE.H, regardless of the name of the resource file (e.g., MYAPP.RC). The **Symbol Header File:** section in the **Resource Includes** dialog in Visual C++, lets you change the name of this header file. Enter a new file name in the section's edit box.
+
+> [!NOTE]
+> resource files not located in the same directory as the .RC file must prepend a relative path with escaped-'\\' to be read properly.
```

Comment:
resource files not located in the same directory as the .RC file must prepend a relative path with respect to the project directory to be read properly.

Reply:
Can you remove "to be read properly" We try to be succinct and if we are telling them to do something, it's so it will work properly.

Reply:
Yes. Also, I adjusted the Note text so it is specific to the section at hand. The text becomes:

[!NOTE]
If the compile-time directive file is not in the same directory as the project, pre-pend the new `MYDIRECTIVES.RC`  file name with a relative path.